src/dependabot/output.ts

```diff
@@ -41,6 +42,7 @@ export function set (updatedDependencies: Array<updatedDependency>): void {
   core.info(`outputs.previous-version: ${prevVersion}`)
   core.info(`outputs.new-version: ${newVersion}`)
   core.info(`outputs.compatibility-score: ${compatScore}`)
+  core.info(`outputs.maintainer-change: ${maintainerChanges}`)
```

Comment:
For consistency, keeping this output field matching the section heading name from the :dependabot: PR's:
```suggestion
  core.info(`outputs.maintainer-changes: ${maintainerChanges}`)
```

For example, see how `dependencyNames` maps to `dependency-names` in line 30:
```
core.info(`outputs.dependency-names: ${dependencyNames}`)
```